client/js/app.js

```diff
@@ -58,19 +58,17 @@ const init = () => {
   const backToTopEl = document.querySelector('.js-back-to-top');
   const toggleShowHideEls = document.querySelectorAll('.js-show-hide');
   const scrollToInfoEls = document.querySelectorAll('.js-scroll-to-info');
-  const workMediaEls = document.querySelectorAll('.js-work-media');
   const copyUrlEls = document.querySelectorAll('.js-copy-url');
   const searchBoxEls = document.querySelectorAll('.js-search-box');
   const iframeContainerEls = document.querySelectorAll('.js-iframe-container');
+  const fullscreenViewers = document.querySelectorAll('.js-image-viewer-fullscreen');
```

Comment:
Just wondering, what's better, `fullscreenViewer`, or `viewerFullscreen`?
We seem to have both?